cli/smartcontract/testdata/nex/nex.go

```diff
@@ -50,3 +75,157 @@ func (c *ContractReader) GetOwner() (util.Uint160, error) {
 func (c *ContractReader) TotalMinted() (*big.Int, error) {
 	return unwrap.BigInt(c.invoker.Call(Hash, "totalMinted"))
 }
+
+// ChangeMinter creates a transaction invoking `changeMinter` method of the contract.
+// This transaction is signed and immediately sent to the network.
+// The values returned are its hash, ValidUntilBlock value and error if any.
+func (c *Contract) ChangeMinter(newMinter *keys.PublicKey) (util.Uint256, uint32, error) {
```

Comment:
Not related to this PR, but we probably need to change the logic of our NEX, because it has Hash160 as an argument of `changeMinter` and as a result of `getMinger`: https://github.com/morphbits/nex-contracts/blob/b6a8309401c2abc83dd28c5d7b4c5c302eb46013/nex/nex.go#L107

Reply:
This was changed purposefully, using keys directly is a bit more convenient for backend operation.